mathphys/evaluation.tex

```diff
@@ -4,7 +4,7 @@ \section{Lehrevaluation}
 
 
 
-\noindent Neben dem \hyperref[kummerkasten]{Kummerkasten} hast Du gegen Ende des Semesters mit der \emph{Evaluation} noch eine Möglichkeit, Deine Profs und Tutorinnen zu bewerten. Das läuft dabei so ab, dass jemand von der Fachschaft (in der Physik) oder von der ZUV (in der Info und Mathe) in die Vorlesung kommt, eine kleine Einleitung gibt und dann einen Bogen austeilt, den Du dann ausfüllst. In den Tutorien der Info und Mathe übernehmen die Tutorinnen die Evaluation. Das ganze dient zweierlei Zwecken: Zum Einen werden besonders schlechte (oder gute) Vorlesungen mit den Dozentinnen nachbesprochen um das Lehrniveau zu steigern bzw. zu halten. Zum Anderen können Dir die Ergebnisse auch helfen, die richtigen Vorlesungen oder Übungsgruppen zu wählen. Wenn die Dozentin ihr Einverständnis zur Veröffentlichung gegeben hat, dann findest Du die Ergebnisse der Physik der letzten Semester z.B. am Fachschaftsraum (im Infoständer neben den Zettelkästen im ersten Stock), sowie im \gls{KIP}, im Hörsaalgebäude der Physik (\Gls{INF} 308) und am Philosophenweg, jeweils an Infobrettern.
+\noindent Neben dem \hyperref[kummerkasten]{Kummerkasten} hast Du gegen Mitte des Semesters mit der \emph{Evaluation} noch die Möglichkeit, deinen Profs und Tutorinnen Feedback zu geben. Das läuft so ab, dass jemand von der Fachschaft (in der Physik) oder Assisteninnen des Profs (in der Info und Mathe) in die Vorlesung kommt, eine kleine Einleitung gibt und einen Bogen austeilt, den Du dann ausfüllst. In den Tutorien der Info und Mathe übernehmen die Tutorinnen die Evaluation, in der Physik ist die Tutorienevaluation Teil des Vorlesungsevaluationsbogen. Das Ganze dient zweierlei Zwecken: Die Studienkommissionen bekommen Einsicht in die Evaluationsergebnisse, sodass besonders schlechte (oder gute) Veranstaltungen mit den Dozentinnen nachbesprochen um das Lehrniveau zu steigern bzw. zu halten. In der Physik wird auch regelmäßig ein Lehrpreis vergeben \footnote{\url{https://mathphys.fsk.uni-heidelberg.de/w/lehrpreise/}}. Zum Anderen werden in der Physik die Ergebnisse auch veröffentlicht (falls die Dozentin Ihre Einverständnis gegeben hat), sodass du nach passenden Vorlesungen und Tutorien suchen kannst. Du findest die Ergebnisse der Physik der letzten beiden Semester z.B. im Fachschaftsraum, sowie im \gls{KIP}-Foyer, im Arbeitsraum und am Philosophenweg am Infobrett im Treppenhaus.
```

Comment:
In dem einen Satz fehlen ein Wort und ein Komma, z. B. "... , sodass besonders schlechte (oder gute) Veranstaltungen mit den Dozentinnen nachbesprochen **werden,** um das Lehrniveau zu steigern bzw. zu halten.